lib/web/scripts/install_node.go

```diff
@@ -46,3 +52,52 @@ exit 1
 var installNodeBashScript string
 
 var InstallNodeBashScript = template.Must(template.New("nodejoin").Parse(installNodeBashScript))
+
+// DBServiceConfig is partial configuration of the Teleport config file.
+// It only contains the top level `db_service` field.
+type DBServiceConfig struct {
+	DBService DBService `yaml:"db_service,omitempty"`
+}
+
+// DBService contains the configurable fields in the context of install node script generation.
+type DBService struct {
+	Enabled          string            `yaml:"enabled"`
+	ResourceMatchers []ResourceMatcher `yaml:"resources"`
+}
+
+// ResourceMatcher has a set of labels used to match the resources to be monitored by the database agent.
+type ResourceMatcher struct {
+	// Labels match resource labels.
+	Labels map[string]utils.Strings `yaml:"labels"`
+}
```

Comment:
Agreed :+1: 
I ended up not using the wider Teleport config because of a cyclic import.

I started this by only adding the labels as templating but I was afraid to mess up some string to yaml conversion along the way.
I'll try your approach using a for loop and see how it goes

Reply:
Just changed it :+1: